src/ucs/arch/cpu.c

```diff
@@ -21,6 +21,8 @@
 #define UCS_CPU_CACHE_TYPE_FILE  "type"
 #define UCS_CPU_CACHE_SIZE_FILE  "size"
 
+#define UCS_CPU_EST_BCOPY_BW_DEFAULT (7000 * UCS_MBYTE)
```

Comment:
seems like we changed UCS to 7000 instead of 5800, is this really the intention? AFAIU we need to change UCP estimations

Reply:
Yes. We updated BW in UCS, and added table for proto v1 in UCP (https://github.com/openucx/ucx/pull/8710#discussion_r1027244727).